dev/scripts/checkshade.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -o pipefail
+set -o nounset   # exit the script if you try to use an uninitialised variable
+
+SHADED_JAR_PATH=${1}
+echo "checkShaded:"${SHADED_JAR_PATH}
+if [ ! -f "${SHADED_JAR_PATH}" ]; then
+    echo "check failed, file is not exist."
+    exit -1
```

Comment:
non-zero exit code is sufficient, negative codes are not common / needed
```suggestion
    exit 1
```